### DISCUSSION TITLE
v2.7.6.1

### DISCUSSION THREAD

**Discussion by Limraj** [SCADA-LTS/Scada-LTS#2736] [General]
## 

The changelist is listed below:

### 📄️ GitHub issues/pullrequest list overview:
1. #2673 Fixed unavailability of acknowledge for active event
2. #2674 Corrected configuration script security context
3. #2675 Fixed longer time startup application
4. #2676 Fixed saving a new graphical view when a validation error occurs
5. #2677 Fixed longer Graphical View save time
6. #2684 error in the report when writing message in the alarm
7. #2686 Fixed SandboxWrapFactory for executing scripts by Rhino
8. #2694 Work-items API extension 
9. #2698 Corrected cache pending events
10. #2706 Fixed save Graphical View with Chart Comparator
11. #2714 Fixed Set point detector in No change detector
12. #2715 Fixed DNP3 Serial data source causing SystemException
13. #2719 Fixed migration from ScadaBR
14. #2724 DataSource Create Button/Option not show
15. #2728 Fixed Import project system exception

<hr /><em>This discussion was created from the release <a href='https://github.com/SCADA-LTS/Scada-LTS/releases/tag/v2.7.6.1'>v2.7.6.1</a>.</em>

**Comment by GabrielTec251** [comment]
I'm having problems implementing scripts involving java classes, it's giving errors with some serotonin functions

**Reply by fabiodurao** [reply]
Gabriel, please detail what the scripts are and what errors you are facing.

**Comment by GabrielTec251** [comment]
[erro.pdf](https://github.com/SCADA-LTS/Scada-LTS/files/13170243/erro.pdf)

there is an error on line 98, I can't paste this data from the database

**Reply by GabrielTec251** [reply]
var eventDao = new com.serotonin.mango.db.dao.EventDao();

**Reply by Limraj** [reply]
The eventDao class has been removed, it has the same functionality as org.scada_lts.mango.service.EventService. Generally, classes ending in Dao will disappear from the code over time, you should use those ending in Service. We also do not recommend using java classes in scripts, due to this type of problems. The code is refactored and improved. 
Configuration shouldn't be so tied to the source code.

**Reply by RenatoExpert** [reply]
@Limraj 
What do you recomend to be used in place of java classes?

**Reply by Limraj** [reply]
Hi @RenatoExpert,
we do not recommend using java classes in scripts..

**Comment by lclaros85** [comment]
Hi guys, I'm new to this project, I have Scadabr 1.0 at work with several graphical representations, along with MySQL, how do I import it into ScadaLTS or 1.2 because we're thinking about leaving it on Ubuntu but on a virtual machine, I'm using the translator from Google, sorry, I'm going to attach an image of one of the graphical representation screens, I've already carried out the entire process of importing the images into the upload and graphic folders in ScadaLTS, the "skeleton" of the representations appears but it doesn't accept the Background, and in the ScadaBR 1.2 nothing appears, not even the skeleton

**Reply by Limraj** [reply]
Hi @lclaros85,
although you didn't provide any details so it's impossible to diagnose your problem in any way, you can try the new version:
https://github.com/SCADA-LTS/Scada-LTS/releases/tag/v2.7.7.1

Regards,
Kamil Jarmusik
